gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -379,9 +375,18 @@ private static ByteString getFooterContent(
     }
   }
 
+  // Estimates a read time out based on read speeds
+  static long getReadTimeoutMillis(GoogleCloudStorageReadOptions readOptions, long objectSize) {
```

Comment:
Depending on how many times this function is called for an object, does it makes sense to initialize this value in the init itself? This function pretty much depends on the object size, and that is not going to change in the lifetime of `this object` right?

Reply:
Makes sense. Moving to instance var